skimage/morphology/tests/test_skeletonize_3d.py

```diff
@@ -175,7 +175,7 @@ def test_two_hole_image():
 
 def test_3d_vs_fiji():
     # generate an image with blobs and compate its skeleton to
-    # the skeleton generated by FIJI
+    # the skeleton generated by FIJI (Plugins>Skeleton->Skeletonize)
```

Comment:
```suggestion
    # the skeleton generated by FIJI (Plugins > Skeleton > Skeletonize)
```
or "(Plugins>Skeleton>Skeletonize)" or "(Plugins->Skeleton->Skeletonize)"